src/containers/Transactions/transaction.scss

```diff
@@ -2,15 +2,18 @@
 
 .transaction {
   position: relative;
+  max-width: 912px;
   min-height: 600px;
-  margin: 0 auto;
 
   @include for-size(tablet-landscape-up) {
```

Comment:
Why do these have the name `up` in them? (Is it for vertical views? Or does it mean "From this size up"?)

Reply:
It is from that size "up" it is something defined here https://github.com/ripple/explorer/blob/staging/src/containers/shared/css/variables.scss#L144-L166